nrpy/infrastructures/BHaH/MoLtimestepping/RK_Butcher_Table_Dictionary.py

```diff
@@ -203,8 +203,8 @@ def generate_Butcher_tables(
     Butcher_dict['AHE'] = (
     [[0],
     [1, 1],
-    ["", sp.Rational(1,2), sp.Rational(1,2)],
-    ["", 1, 0]]
+    ["HO", sp.Rational(1,2), sp.Rational(1,2)],
```

Comment:
Please add a comment describing what HO and LO stand for.

Reply:
Is it sufficient to put it in the class docstring since the comment is related to all adaptive schemes?

Reply:
I'd add a longer comment in the docstring and short one-line comments below (even if all they do is refer to the docstring). This is just in case a user is focused on the dictionary entry and doesn't think to look at the docstring.